src/operations/expansions/area.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Calculate the area under the given points
+ */
+const area = (points) => {
```

Comment:
nice !
are points just arrays of vec2s ? in that case is this not maths=> vec2 function ?

Reply:
Possibly, but vec2 should probably stay simple, like compare(a, b)

I think this could moved to math/utils but wanted to know if there's value. It's a generic function for any number of 2D vectors. It could even go into measurements, but feels more like an internal utility function.